recipe/meta.yaml

```diff
@@ -169,10 +170,12 @@ about:
   license_file: LICENSE.txt
   summary: MNE-Python is a software for MEG and EEG data analysis.
   description: |
-    `mne-base` contains the basic functionality for data analysis and 2D
-    visualization. For full functionality, including 3D visualization,
-    use the `mne` package instead. `mne-installer-menus` is only used by the
-    standalone installers and should not be installed manually.
+    The main package for MNE-Python is named `mne` in conda-forge and should
+    be suitable for most users. The conda recipe produces the following:
+
+    - `mne`: should be installed for full functionality including 3D visualization.
+    - `mne-base`: only pulls dependencies for basic functionality and 2D visualization.
+    - `mne-installer-menus`: should not be installed manually, as it is only meant to be used by the standalone installers.
```

Comment:
@hoechenberger I refactored this since I think the description on our main MNE page is confusing in terms of how it presents information:

https://anaconda.org/conda-forge/mne

Do you agree this is clearer / would be better for people to see when landing on the page above?

Reply:
Okay the re-render did it :)

Reply:
LGTM!